lms/djangoapps/course_home_api/course_metadata/v1/views.py

```diff
@@ -104,7 +98,8 @@ def get(self, request, *args, **kwargs):
             'is_self_paced': getattr(course, 'self_paced', False),
             'is_enrolled': user_is_enrolled,
             'can_load_courseware': can_load_courseware,
-            'celebrations': celebrations,
+            'celebrations': courseware_meta.celebrations,
+            'verified_mode': courseware_meta.verified_mode,
```

Comment:
verified mode is also present in the outline view, but is necessary here for the other tabs.  
should I remove it from the outline view and migrate uses to use this field?
only issue is that if we don't implement this experiment we'd need to remove this field and add that one back

Reply:
I might prefer a use of VerifiedModeSerializerMixin, like the other course home tab views, instead of reaching into a another view's attribute. Same underlying definition (both call `serialize_upgrade_info`), but just a cleaner separation.

I like the idea of migrating uses to this field, yeah. But I think you make a good point about potentially not implementing this experiment. So let's save that for the experiment cleanup. But I would like to see it done if we do keep this.

Reply:
Ok will change to VerifiedModeSerializerMixin